docs/source/refs.bib

```diff
@@ -27,11 +27,19 @@ @inproceedings{hillmichExlpoitingQuantumTeleportation2021
     url = {https://www.cda.cit.tum.de/files/eda/2021_aspdac_exploiting_teleportation_in_quantum_circuit_mappping.pdf},
 }
 
-@inproceedings{burgholzerRandomStimuliGeneration2021,
+@inproceedings{burgholzer2022limitingSearchSpace,
```

Comment:
Thanks for catching that. Typical c&p error 🙃

Reply:
Happens to the best.